src/ui/common/src/components/workflows/artifact/metric/history.tsx

```diff
@@ -49,16 +49,24 @@ const MetricsHistory: React.FC<Props> = ({ historyWithLoadingStatus }) => {
     schema: metricHistorySchema,
     data: (historyWithLoadingStatus.results?.results ?? []).map(
       (artifactStatusResult) => {
+        let timestamp = new Date(
```

Comment:
Is comparing with 'Invalid Date' the standard way to determine if date is valid? I'm considering the following:
```
const finished_at = ...
const timestamp = !!finished_at ? new Date(finished_at).toLocaleString() : "Unknown"
```

Reply:
yeah I think we can do that instead.

Reply:
yeah I think we can do that instead. Will make a linear task to get these fixes in.